blockem/blockem.php

```diff
@@ -4,49 +4,52 @@
-	Addon::registerHook('conversation_start', 'addon/blockem/blockem.php', 'blockem_conversation_start');
-	Addon::registerHook('item_photo_menu', 'addon/blockem/blockem.php', 'blockem_item_photo_menu');
-	Addon::registerHook('enotify_store', 'addon/blockem/blockem.php', 'blockem_enotify_store');
+	Addon::registerHook('display_item'               , 'addon/blockem/blockem.php', 'blockem_display_item');
+	Addon::registerHook('addon_settings'             , 'addon/blockem/blockem.php', 'blockem_addon_settings');
+	Addon::registerHook('addon_settings_post'        , 'addon/blockem/blockem.php', 'blockem_addon_settings_post');
+	Addon::registerHook('conversation_start'         , 'addon/blockem/blockem.php', 'blockem_conversation_start');
+	Addon::registerHook('item_photo_menu'            , 'addon/blockem/blockem.php', 'blockem_item_photo_menu');
+	Addon::registerHook('enotify_store'              , 'addon/blockem/blockem.php', 'blockem_enotify_store');
 }
 
 function blockem_uninstall()
 {
 	Addon::unregisterHook('prepare_body_content_filter', 'addon/blockem/blockem.php', 'blockem_prepare_body_content_filter');
-	Addon::unregisterHook('prepare_body', 'addon/blockem/blockem.php', 'blockem_prepare_body');
-	Addon::unregisterHook('display_item', 'addon/blockem/blockem.php', 'blockem_display_item');
-	Addon::unregisterHook('addon_settings', 'addon/blockem/blockem.php', 'blockem_addon_settings');
-	Addon::unregisterHook('addon_settings_post', 'addon/blockem/blockem.php', 'blockem_addon_settings_post');
-	Addon::unregisterHook('conversation_start', 'addon/blockem/blockem.php', 'blockem_conversation_start');
-	Addon::unregisterHook('item_photo_menu', 'addon/blockem/blockem.php', 'blockem_item_photo_menu');
-	Addon::unregisterHook('enotify_store', 'addon/blockem/blockem.php', 'blockem_enotify_store');
+	Addon::unregisterHook('prepare_body'               , 'addon/blockem/blockem.php', 'blockem_prepare_body');
+	Addon::unregisterHook('display_item'               , 'addon/blockem/blockem.php', 'blockem_display_item');
+	Addon::unregisterHook('addon_settings'             , 'addon/blockem/blockem.php', 'blockem_addon_settings');
+	Addon::unregisterHook('addon_settings_post'        , 'addon/blockem/blockem.php', 'blockem_addon_settings_post');
+	Addon::unregisterHook('conversation_start'         , 'addon/blockem/blockem.php', 'blockem_conversation_start');
+	Addon::unregisterHook('item_photo_menu'            , 'addon/blockem/blockem.php', 'blockem_item_photo_menu');
+	Addon::unregisterHook('enotify_store'              , 'addon/blockem/blockem.php', 'blockem_enotify_store');
 }
 
-function blockem_addon_settings(&$a, &$s)
+function blockem_addon_settings (App $a, &$s)
 {
-
-	if(! local_user())
+	if (! local_user()) {
 		return;
+	}
 
     /* Add our stylesheet to the page so we can make our settings look nice */
-
     $a->page['htmlhead'] .= '<link rel="stylesheet"  type="text/css" href="' . $a->get_baseurl() . '/addon/blockem/blockem.css' . '" media="all" />' . "\r\n";
```

Comment:
Indentation looks wrong (space instead of tabs)

Reply:
A lot lines will then be changed, if that is okay, I start changing?

Reply:
Yes, as long as it’s limited to a single addon 😉